designer/server/src/common/components/service-header/template.njk

```diff
@@ -1,3 +1,5 @@
+{% from "service-header/environment-tag.njk" import envTag %}
```

Comment:
Looking good @KiranVarma02

Probably want to split this out from the service header if it's going in the GOV.UK header too?

I'd prefix it with "app" as `appTagEnv()` like the others, and use the `macro.njk` + `template.njk` filenames

Then I think it's job done

```suggestion
{% from "tag-env/macro.njk" import appTagEnv %}
```

Reply:
done, I like it. Thanks @colinrotherham